homeassistant/components/smhi/config_flow.py

```diff
@@ -0,0 +1,138 @@
+"""Config flow to configure smhi component.
+
+First time the user creates the configuration and
```

Comment:
This information is not relevant any more

Reply:
Yes after my change to default values in form that is correct, thanks !